docs/architecture/microservices/docker-application-development-process/docker-app-development-workflow.md

```diff
@@ -47,21 +47,21 @@ To begin, make sure you have [Docker Community Edition (CE)](https://docs.docker
 
 [Get started with Docker CE for Windows](https://docs.docker.com/docker-for-windows/)
 
-In addition, you need Visual Studio 2019 version 16.8, with the **.NET Core cross-platform development** workload installed, as shown in Figure 5-2.
+In addition, you need Visual Studio 2022 version 17.0, with the **.ASP.NET and web development** workload installed, as shown in Figure 5-2.
```

Comment:
In the earlier line the link has the title "Docker CE for Windows". Can you change that to Docker Desktop for Windows?

Reply:
Yes, changed it.